pulsar/internal/rpc_client.go

```diff
@@ -90,7 +90,7 @@ func (c *rpcClient) Request(logicalAddr *url.URL, physicalAddr *url.URL, request
 		*RPCResult
 		error
 	}
-	ch := make(chan Res)
+	ch := make(chan Res, 1)
```

Comment:
```suggestion
	ch := make(chan Res, 10)
```

Normally we use 10 as the buffer size.